app/services/csv_exporter.rb

```diff
@@ -39,30 +39,30 @@ def initialize(kase)
   def to_csv
     begin
       [
-          @kase.number,
-          @kase.decorate.pretty_type,
-          @kase.current_state,
-          @kase.responding_team&.name,
-          @kase.responder&.full_name,
-          @kase.received_date&.strftime('%F'),
-          @kase.internal_deadline&.strftime('%F'),
-          @kase.external_deadline&.strftime('%F'),
-          @kase.date_responded.present? ? @kase.date_responded.strftime('%F') : nil,
-          @kase.workflow,
-          @kase.name,
-          @kase.requester_type,
-          dequote_and_truncate(@kase.message),
-          @kase.info_held_status&.name,
-          @kase.outcome&.name,
-          @kase.refusal_reason&.name,
-          @kase.exemptions.map{ |x| CaseClosure::Exemption.section_number_from_id(x.abbreviation) }.join(','),
-          @kase.postal_address,
-          @kase.email,
-          @kase.appeal_outcome&.name,
-          @kase.respond_to?(:third_party) ? @kase.third_party : nil,
-          @kase.respond_to?(:reply_method) ? @kase.reply_method : nil,
-          @kase.respond_to?(:subject_type) ? @kase.subject_type : nil,
-          @kase.respond_to?(:subject_full_name) ? @kase.subject_full_name : nil
+        @kase.number,
+        @kase.decorate.pretty_type,
+        I18n.t("state.#{@kase.current_state}").downcase,
+        @kase.responding_team&.name,
+        @kase.responder&.full_name,
+        @kase.received_date&.strftime('%F'),
+        @kase.flagged? ? @kase.internal_deadline&.strftime('%F') : nil,
```

Comment:
should this not be '' (blank) rather than nil in a CSV?

Reply:
Nil is what is currently being used such as
`@kase.respond_to?(:third_party) ? @kase.third_party : nil`
is there a need to change it to " "?